### PR TITLE
feat/add-icq-relayer-shutdown-description

### PR DESCRIPTION
**Pull request by sotnikov-s**
inspired during work on https://github.com/neutron-org/neutron-query-relayer/pull/50

### CODE REVIEW

**Review by foxpy — DISMISSED** [review]

File: docs/relaying/icq-relayer.md [hunk 1]
```diff
@@ -132,3 +132,7 @@ Notes:
 ```
 docker run --env-file .env.example -v /Users/your-user/.neutrond:/keyring -p 9999:9999 neutron-org/neutron-query-relayer
 ```
+
+## Shutting the Relayer down
+
+During the execution the Neutron ICQ Relayer receives events from Neutron, reads remote chain's state, and modifies its own state and the Neutron' one. In order to reach a reliable and consistent flow the Relayer is designed the way it finishes initialised interactions with its local storage on received `SIGINT` and `SIGTERM`. It usually takes a fraction of a second.
```
Comment (foxpy) [inline docs/relaying/icq-relayer.md:138]:
```suggestion
During the execution the Neutron ICQ Relayer receives events from Neutron, reads remote chain's state, and modifies its own state and the Neutron' one. In order to reach a reliable and consistent flow the Relayer is designed the way it finishes initialised interactions with its local storage when `SIGINT` or `SIGTERM` signal is received. It usually takes a fraction of a second.
```

**Review by foxpy — CHANGES_REQUESTED** [review]

File: docs/relaying/icq-relayer.md (hunk 1 above)
Comment (sotnikov-s) [reply docs/relaying/icq-relayer.md:138]: what's wrong with the way I wrote it? :)

Comment (NeverHappened) [reply docs/relaying/icq-relayer.md:138]: I would just change to `is designed **to finish** initialised ....`

Comment (foxpy) [reply docs/relaying/icq-relayer.md:138]: > what's wrong with the way I wrote it? :)

Just suggesting you a little cleaner way to write it.

Actually, we need someone who is at least C1 in English (I guess almost everyone in our team is B1-B2) to proof read our documentation and point out how to write it cleaner.

You are free to close this thread yourself, and, if you do so, I will approve this PR.

**Review by foxpy — APPROVED** [review]
Approving it since it is stalled…

**Pull request merged**